ibm/mas_devops/roles/cp4d_service/tasks/wait/wait-ccs.yml

```diff
@@ -104,10 +104,15 @@
 
   when: not is_ccs_already_patched
 
-- include_tasks: "tasks/wait/wait-elasticsearch.yml"
+- include_tasks: "tasks/wait/wait-opensearch.yml"
```

Comment:
What's the impact with elasticsearch/opensearch difference in 5.1 vs 5.2 for upgrade scenario?  Does ES get automatically replaced by OS?